### PR TITLE
Quick update of topics UI

### PR DESCRIPTION
**Pull request by taniki**
- no more carrousel
- no more CTA
- move related tags upward

### CODE REVIEW

**Comment by taniki** [comment]
commandé de Laure

**Comment by abulte** [comment]
screenshots? 🙏

**Comment by taniki** [comment]
image

**Comment by noirbizarre** [comment]
Seems OK? did you check the mobile rendering ?

**Comment by noirbizarre** [comment]
THe call to action has been removed but historically it has been added because people were asking where it was.
Those pages wil become the only ones without quick access to contribution. Did we check the button usage (ie. are we sure the button wasn't used before the removal) ?

**Comment by taniki** [comment]
image

**Comment by noirbizarre** [comment]
The topic is header is fine but the cards rendering is broken :/

I already had this issue, a class is missing (I think the home cards already have the fix)

File: theme/less/gouvfr-topic.less [hunk 1]
```diff
@@ -9,6 +9,11 @@ body.topic-display {
         font-size: 14px;
         line-height: 1.6em;
 
+        a {
```
Comment (noirbizarre) [inline theme/less/gouvfr-topic.less]: :+1:

Comment (taniki) [reply theme/less/gouvfr-topic.less]: :+1:

**Pull request merged**